src/detection_utils/boxes.py

```diff
@@ -142,8 +148,8 @@ def generate_targets(
         return targets_cls, targets_reg
 
     ious = box_overlaps(anchor_boxes, truth_boxes)  # NxK
-    max_ious = ious.max(axis=1)                     # N IoUs
-    max_idxs = ious.argmax(axis=1)                  # N indices
+    max_ious = ious.max(axis=1)  # N IoUs
```

Comment:
is there a way to keep comments lined up? there was a reasons we had all those spaces